File: 11_backbone/burningAirlinesFinal/app/assets/javascripts/models/Plane.js
var app = app || {};

app.Plane = Backbone.Model.extend({
  urlRoot: "/planes",
  default: {
    name: "",
    rows: "",
    columns: ""
  },
  initialize: function() {
    console.log("a new plane was create")
  }
})


  //
  // create_table "planes", force: :cascade do |t|
  //   t.text    "name"
  //   t.integer "rows"
  //   t.integer "columns"
  // end

File: 08 Rails/bacmd/app/models/plane.rb
# == Schema Information
#
# Table name: planes
#
#  id      :integer          not null, primary key
#  name    :text
#  rows    :integer
#  columns :integer
#

class Plane < ActiveRecord::Base
  has_many :flights
  has_many :reservations, :through => :flights
end
